pages/instructorQuestion/instructorQuestion.sql

```diff
@@ -19,12 +19,106 @@ WHERE
     AND aaci.authorized;
 
 -- BLOCK select_question
+WITH num_attempts_histogram AS (
+    SELECT
+        aq.question_id,
+        histogram(number_attempts, 1, 10, 10) as num_attempts_histogram,
```

Comment:
Because we aren't storing any question statistics in this PR. We are storing instance question statistics and assessment question statistics but no question statistics. That is part of the "next" commit.